packages/backend/src/dbt/dbtCliClient.ts

```diff
@@ -205,7 +205,8 @@ export class DbtCliClient implements DbtClient {
             op: 'dbt',
             description: 'test',
         });
-        await this._runDbtCommand('debug');
+        await this.installDeps();
+        await this._runDbtCommand('parse');
```

Comment:
We no longer run `dbt debug` which checks the warehouse connection. That's because the spark connection profile for dbt is a bit different to what we use to connect. So we just check that we can parse the project, this requires the project dependencies.

Reply:
@ZeRego ^ what do you think?

I feel like when we load a project, we're testing that we can clone the project and it has a dbt_project.yml